### ISSUE TITLE
Print from Stdin

### ISSUE THREAD

**Issue by bernadinm** [risor-io/risor#72]
I am looking to know if the following is capable?

## Printing from STDIN Python

```bash
 echo test | python -c "import sys; print(sys.stdin.read().strip())"
test
```

## Printin from STDIN in risor
```bash
 echo test |  risor -c `...` # what goes here
```

## Printin from STDIN in risor
```bash
 echo test |  risor -c `...` # what goes here
```

## Other MISC examples
```bash
❯ echo test | risor -c 'print("hello")
hello
```

**Comment by myzie** [comment]
Hi @bernadinm, thanks for asking about this. I had neglected working on any useful stdin support, as silly as that sounds.

I just took a first cut at it here: https://github.com/risor-io/risor/pull/73

Can you review the examples in the PR and let me know how it looks to you?

**Comment by myzie** [comment]
This is available in Risor as of release 0.10.0. Let me know if you have any trouble with it!
https://github.com/risor-io/risor/releases/tag/v0.10.0
